develop/components/pages/CheckBoxPage.tsx

```diff
@@ -1,62 +1,74 @@
-                        </div>
-                    </div>
+const docMD: string = require("../../../src/CheckBox/readme.md");
+
+const CheckBoxPage: React.FunctionComponent = () => {
+    const [checkbox1, setCheckbox1] = React.useState<boolean>(true);
+    const [checkbox2, setCheckbox2] = React.useState<boolean>(false);
+    const [checkbox3, setCheckbox3] = React.useState<boolean>(true);
+    const [tabValue, setTabValue] = React.useState<number>(0);
+
+    const [input, setInput] = React.useState<string>("");
+
+    return (
+        <div className="route-template container">
+            <div className="info-holder">
 
-                    <div className="info">
-                        <h2>Output</h2>
-                        <p>Here are few checkboxes with different configurations:</p>
-                        <div className="result">
-                            <CheckBox
-                                name="checkbox-1"
-                                label="Checkbox 1"
-                                checked={this.state.checkbox1}
-                                onChange={(event) => { this.setState({ checkbox1: event.target.checked }); }}
-                            />
-                            <CheckBox
-                                name="checkbox-2"
-                                label="Checkbox 2"
-                                checked={this.state.checkbox2}
-                                onChange={(event) => { this.setState({ checkbox2: event.target.checked }); }}
-                                description="Some description"
-                            />
-                            <CheckBox
-                                name="checkbox-3"
-                                label="Checkbox 3"
-                                checked={this.state.checkbox3}
-                                onChange={(event) => { this.setState({ checkbox3: event.target.checked }); }}
-                                disabled={true}
-                                description="Disabled"
-                            />
-                        </div>
+                <div className="info">
+                    <div className="md-file">
+                        <Highlight innerHTML={true}>{docMD}</Highlight>
                     </div>
+                </div>
 
+                <div className="info">
+                    <h2>Output</h2>
+                    <p>Here are few checkboxes with different configurations:</p>
+                    <div className={"result" + (tabValue === 2 ? " wide" : "")}>
+                        <Tabs
+                            list={[
+                                { text: "Normal" },
+                                { text: "Condensed" },
+                                { text: "Inline" }
+                            ]}
+                            activeTab={tabValue}
+                            onClick={(index: number) => setTabValue(index)}
```

Comment:
you can just do this:
```suggestion
                            onClick={setTabValue}
```

Reply:
Done